great_expectations/core/expectation_suite.py

```diff
@@ -739,9 +739,19 @@ def add_expectation(
             overwrite_existing=overwrite_existing,
         )
 
+    @public_api
     def show_expectations_by_domain_type(self) -> None:
-        """
-        Displays "ExpectationConfiguration" list, grouped by "domain_type", in predetermined designated order.
+        """Displays "ExpectationConfiguration" list, grouped by "domain_type", in predetermined designated order.
+
+        The means of displaying is through the use of the "Pretty Print" library method "pprint.pprint()".
+
+        Args: N/A
+
+        Returns:
+            N/A
+
+        Raises:
+            N/A
```

Comment:
Don't think we need any of these

Reply:
Agreed. I think the `linter` should be failing on this too.

Reply:
@cdkini Fixed -- thanks.

Reply:
@Shinnnyshinshin Fixed -- thanks.